site/src/components/popover.md

```diff
@@ -75,3 +75,24 @@ Popover is left aligned by default, but also can be centered or right aligned re
 	<div class="popover-content">Popover content.</div>
 </details>
 ```
+
+### Vertical alignment
+
+The popover can also show up above the button.
+
+```html
+<details class="popover popover-top margin-xxs">
+	<summary class="button">Popover on top of the button</summary>
+	<div class="popover-content">Popover content.</div>
+</details>
+
+<details class="popover popover-top popover-center margin-xxs">
```

Comment:
Can you split that example into two and say "vertical and horizontal alignment can be used together" in between

Reply:
added